spec/requests/create_item_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'create list item api' do
+  it 'creates a list item' do
+
+    user = create(:user)
+    api_key = user.api_key
+    list1 = create(:list, :title => 'List 1', :user => user)
+
+    post "/api/v1/lists/#{list1.id}/items", {"item" => {"name" => "item 1"}}, "X-Api-Key" => api_key
+
+    expect(response.status).to eq(200)
```

Comment:
201 for "created" would be a more accurate response code for a successful create request.